modules/azure/core/variables.tf

```diff
@@ -97,3 +97,14 @@ variable "unique_suffix" {
   description = "Unique suffix that is used in globally unique resources names"
   type        = string
 }
+
+variable "notification_email" {
+  description = "Email address to send alert notifications"
+  type        = string
+}
```

Comment:
Since the notification_email don't have any default value we have to define it.
Not a problem in our templates repo though.

We can of course add it and set it to something like `""` but if we do then we might make a human error and not sending the errors to anything. 

We could also add our own email in here but we haven't done so in other locations so I guess we are trying not to spread it to much.